packages/aurora-feed/index.js

```diff
@@ -1,10 +1,33 @@
 import React from "react";
-import { Card } from "../aurora-ui";
-import { CardAtBottomEditor } from "../aurora-editor";
+import Note from "../aurora-note";
+import FeedEditor from "./FeedEditor.js";
 import search from "../aurora-search";
 import styled from "styled-components";
+import uuidv4 from "uuid";
+import { EditorState } from "draft-js";
+import { save, loadNotes } from "../aurora-save";
 
-const TUTORIAL_MESSAGES = [{ text: "Press Shift+Enter to add a new note." }];
+/**
+ * Adds a "text" version of the editor state to each note in the notes object
+ */
+const fromNotesToPlainTexts = notes => {
```

Comment:
what is this needed for? Are you trying to attach metadata to the editor states?

Reply:
It converts some notes into plain text objects so that we can search them. I change it to `fromNotesToSearchableObjects` to make that more clear.